src/scss/_variables.scss

```diff
@@ -24,6 +24,9 @@ $error: $fail;
 
 $gradient_dark: linear-gradient(90deg, $blue_dark 0%, #07162c 100%);
 
+
+$font-color: #333333;
```

Comment:
Try and keep variable colour names related to the colour rather than their purpose, it helps to work out which is which. This would probably be `$grey_dark`